packages/@react-aria/selection/src/useTypeSelect.ts

```diff
@@ -37,14 +37,21 @@ export function useTypeSelect(options: TypeSelectOptions): TypeSelectAria {
       return;
     }
 
+    const isTimedOut = state.search === '';
+
     state.search += character;
 
+    const keyBelowFocused = keyboardDelegate.getKeyBelow(selectionManager.focusedKey);
```

Comment:
From the other review comments it sounds like this should be `focusedKey = selectionManger.focusedKey` rather than `keyBelowFocused` right?  I believe we want the initial search to include the currently focused key rather than starting from the key right below the initially focused key.

The test case I was testing locally was if we had something like "CarA", "CarB",  and "CarC" and the user was initially focused on "CarA". The current implementation has the focus move to "CarB" if the user types "C" but the mdn select keeps focus on "CarA" until the user finishes typing "CarB"

Reply:
@devongovett @dannify lemme know if I interpreted your prior comments correctly

Reply:
@LFDanLu - that's how I had it, but it seems like we don't want to replicate MDN's behavior.

Also, if we set `focusedKey = selectionManger.focusedKey`, then we wouldn't be able to navigate through after timeout. So in the "Car" example, you'd key in 'C', which takes you to "CarA", then wait, then key in 'C' again, so it takes you to "CarB", etc. If we want it to stay on "CarA" after timeout, then yes, it should be `selectionManger.focusedKey`.

This is what I understood from @devongovett and @dannify.

Reply:
I'm sorry, I think I may have confused you here @solimant  
I'm looking at the storybook and I don't see that this code has made the functionality any different from before. I think Daniel is right here. We should start searching from (and including) the focused item. So in your example, it would stay on CarA after timeout. This is a deliberate difference from the native behavior.

Reply:
It seems like you have written the solution in your comments already :) 
`focusedKey = selectionManger.focusedKey`
I would expect the tests to need updating when we change this functionality.

Reply:
@dannify - No problem! Glad we reached a consensus 🙂Will make the fix.